src/components/Dao/ProposalSchemeCard.tsx

```diff
@@ -91,17 +91,14 @@ export default withSubscription({
     const arc = getArc();
     const dao = arc.dao(props.dao.address);
     return combineLatest(
-      props.scheme.state(),
+      props.scheme.state(standardPolling()),
       dao.proposals({ where: {
         scheme:  props.scheme.id,
         // eslint-disable-next-line @typescript-eslint/camelcase
         stage_in: [IProposalStage.Boosted, IProposalStage.QuietEndingPeriod],
       },
       orderBy: "boostedAt",
-      }, {
-        fetchAllData: true,
-        subscribe: true, // subscribe to updates of the proposals. We can replace this once https://github.com/daostack/subgraph/issues/326 is done
-      }) // the list of boosted proposals
+      }, { fetchAllData: true }) // the list of boosted proposals
```

Comment:
no need polling here ?

Reply:
Ha, I just added that (see previous comment above)